frontend/src/components/ShootStatus.vue

```diff
@@ -43,8 +43,9 @@ SPDX-License-Identifier: Apache-2.0
         <span v-if="showStatusText" class="d-flex align-center ml-2">{{statusTitle}}</span>
       </div>
       <template v-if="showStatusText">
-        <div v-for="({ description }) in tooltip.errorCodeObjects" :key="description">
-          <div class="font-weight-bold error--text wrap">{{description}}</div>
+        <div v-for="({ description, link }) in tooltip.errorCodeObjects" :key="description">
+          <div class="font-weight-bold error--text wrap-text">{{description}}</div>
+          <div class="font-weight-bold" v-if="link"><external-link :url="link.url">{{link.text}}</external-link></div>
```

Comment:
<img width="1096" alt="Screenshot 2022-05-09 at 17 37 54" src="https://user-images.githubusercontent.com/5526658/167446556-c344fe1e-604b-4b48-84b2-97288e15fa81.png">
I completely overlooked the hint as I didn't saw it in relation to the error message. Maybe because it had no error color or maybe because it was in a newline? 

I also expected to see this hint within the popover but there was none
<img width="1019" alt="Screenshot 2022-05-09 at 17 42 57" src="https://user-images.githubusercontent.com/5526658/167447053-6a63bd09-9965-4e3a-8037-e36f41ddf545.png">